config_test.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"flag"
+	"github.com/decred/dcrd/dcrutil"
```

Comment:
Our convention is to segregate the remote imports from the standard library imports.  i.e.

```go
import (
	"flag"
	"io/ioutil"
	"os"
	"path/filepath"
	"testing"

	"github.com/decred/dcrd/dcrutil"
)
```

Another update for the dev docs?